appinterface/projection/crossfire/crossfire.go

```diff
@@ -30,6 +30,11 @@ type Crossfire struct {
 	phaseTwoStartTime        utctime.UTCTime
 	phaseThreeStartTime      utctime.UTCTime
 	competitionEndTime       utctime.UTCTime
+	JackpotOneStartTime      utctime.UTCTime
```

Comment:
Keep it lowercase to be consistent with other fields?

Reply:
You are right.